### DISCUSSION TITLE
Does not maintain brightness level on screen saver

### DISCUSSION THREAD

**Discussion by scottthomascarter** [MonitorControl/MonitorControl#1702] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Macbook Pro, MacOS Sequoia, external HDMI monitor. App works perfectly and as described, but when the screen saver activates it returns to the hardware brightness setting of the monitor. I don't know if this is a bug or a feature. But it would be a good option to be able to maintain a (dimmed, in my case) setting even with the screen saver active. This, in my case, is because the whole point of me wanting to dim is to not illuminate the room with the monitor when the lights are off.

### Steps to reproduce

n/a

### Expected behavior

n/a

### Anything else?

n/a

### Environment Information (please complete the following information)

```markdown
- macOS version:Sequoia 15.1.1
- Mac model:Macbook Pro, 13 inch, M1, 2020
- MonitorControl version:1.0.0 Build 212
- Monitor(s):Samsung LC27T55
- Apple Silicon/M1 (yes or no): yes
```

**Comment by kn666** [comment]
Similar problem after resolution change or monitor reconnection. Pressing brightness up or down restores current level.

**Comment by waydabber** [comment, marked as answer]
Some displays might forget their DDC setting after sleep. Try changing the "Upon startup or wake" setting under General to fix this. Alternatively, you can switch to BetterDisplay as it has more options regarding this.

**Reply by kn666** [reply]
I use software gamma. Upon "Upon startup or wake" doesn't work after resolution change or monitor reconnection in my case. Does it work for you?

**Reply by waydabber** [reply]
Oh ok. Gamma should not reset after wake but on some Macs it indeed does. This issue should not be present in BetterDisplay with color table control only.
